src/extension/sdk/utils.ts

```diff
@@ -243,43 +243,49 @@ export class SdkUtils {
 				this.logger.info(`Found Fuchsia project that is not vanilla Flutter`);
 		}
 
-		const flutterSdkSearchPaths = [
-			workspaceConfig?.flutterSdkHome,
-			config.flutterSdkPath,
-			// TODO: These could move into processFuchsiaWorkspace and be set on the config?
-			fuchsiaRoot && path.join(fuchsiaRoot, "lib/flutter"),
-			fuchsiaRoot && path.join(fuchsiaRoot, "third_party/dart-pkg/git/flutter"),
-			firstFlutterMobileProject,
-			firstFlutterMobileProject && extractFlutterSdkPathFromPackagesFile(firstFlutterMobileProject),
-			firstFlutterMobileProject && path.join(firstFlutterMobileProject, ".flutter"),
-			firstFlutterMobileProject && path.join(firstFlutterMobileProject, "vendor/flutter"),
-			process.env.FLUTTER_ROOT,
-			isLinux ? "~/snap/flutter/common/flutter" : undefined,
-			"~/flutter-sdk",
-			"~/.flutter-sdk",
-		].concat(paths).filter(notUndefined);
-
-		let flutterSdkResult = this.findFlutterSdk(flutterSdkSearchPaths);
-
-		// Handle the case where the Flutter snap has not been initialised.
-		if (!flutterSdkResult.sdkPath && flutterSdkResult.candidatePaths.includes(snapBinaryPath)) {
-			// Trigger initialization.
-			this.logger.info(`No Flutter SDK found, but a 'flutter' binary did point at ${snapBinaryPath} so attempting to initialize...`);
-			await initializeFlutterSdk(this.logger, snapFlutterBinaryPath);
-
-			// Then search again.
-			this.logger.info(`Snap initialization completed, searching for Flutter SDK again...`);
-			flutterSdkResult = this.findFlutterSdk(flutterSdkSearchPaths);
-		}
+		let flutterSdkPath;
+		if (workspaceConfig.flutterBazel) {
+			hasAnyFlutterProject = true;
+			hasAnyFlutterMobileProject = true;
+			flutterSdkPath = workspaceConfig?.flutterSdkHome;
```

Comment:
Setting these probably gets the Flutter functionality enabled, but there might also need to be some changes to `isInFlutterProject`(?) to trigger the Flutter paths for specific files (eg. when launching projects, or running commands like Get Packages that choose between Dart/Flutter).

Reply:
I'm not sure I understand the context here - I'll try to schedule time with you tomorrow to discuss in real time.